source_code/main_mcu/src/LOGIC/logic_user.c

```diff
@@ -932,38 +933,40 @@ fido2_return_code_te logic_user_get_webauthn_credential_key_for_rp(cust_char_t*
         /* Fetch username for that credential id, username is already 0 terminated by code above */
         logic_database_get_webauthn_username_for_address(child_address, temp_user_name);
         
-        /* If user specified to be prompted for login confirmation */
-        if ((logic_user_get_user_security_flags() & USER_SEC_FLG_LOGIN_CONF) != 0)
+        if ( (flags & FIDO2_GA_FLAG_SILENT) != FIDO2_GA_FLAG_SILENT)
```

Comment:
I think there's an issue here: didn't we want to send a denied operation when the flag is set instead? Something like:
if ( (flags & FIDO2_GA_FLAG_SILENT) == FIDO2_GA_FLAG_SILENT)
{
    return FIDO2_OPERATION_DENIED;
}

before the if()?

Reply:
We want to actual send a valid signed assertion back with UP=0 for the silent GA case so that the platform (WIndows etc) knows that this is the right device (in case of more than 1 device) and that it picked the right credential to use (that it actually exists).